### PR TITLE
Make "My Home" in course nav an anchor

### PR DESCRIPTION
**Pull request by omsmith**

### CODE REVIEW

File: link-highlight-icon.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	Polymer({
+		is: 'd2l-navigation-link-highlight-icon',
+		extends: 'a',
+		properties: {
+			icon: {
+				type: String,
+				reflectToAttribute: true
+			},
+			text: String,
+			title: {
```
Comment (dlockhart) [inline link-highlight-icon.html]: Sucks that it's this difficult to reflect something to a host attribute. Ah well.

Comment (omsmith) [inline link-highlight-icon.html]: Meh - the button one was already doing the same thing, so not really work on my part.

**Comment by dlockhart** [comment]
Hmm, we should maybe not be running sauce if it's a fork eh...

File: link-highlight-icon.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../d2l-offscreen/d2l-offscreen.html">
+<link rel="import" href="button-highlight-styles.html">
+<dom-module id="d2l-navigation-link-highlight-icon">
+	<style include="d2l-navigation-button-highlight-styles">
+		:host([is="d2l-navigation-link-highlight-icon"]) {
```
Comment (omsmith) [inline link-highlight-icon.html]: This was the same trouble @awikkerink was having, so I went with the solution that seemed to work from that comment thread.

**Pull request merged**